### DISCUSSION TITLE
starting_solution issue.

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#3189] [General]
I am trying to do something like[ this](https://docs.pybamm.org/en/latest/source/examples/notebooks/simulating-long-experiments.html#Starting-solution) that is simulation one experiment and use sol1 as starting point for next solution using sim.solve(starting_solution=sol1) however, I get the error 'sol1' not defined. It is showing in variables but i get this error.

**Comment by valentinsulzer** [comment]
This is definitely an issue with your code and not pybamm.
